pam/integration-tests/integration_test.go

```diff
@@ -0,0 +1,112 @@
+package main_test
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/authd/internal/testutils"
+)
+
+var daemonPath string
+
+func TestIntegration(t *testing.T) {
```

Comment:
Would be better to call this (and the whole file) in a more specific way? This is the integration test for the terminal interface only, while further integration tests should be added (ideally in different files) for both GDM and pure-pam interaction (i.e. for ssh and other non-interactive terminals).

Reply:
I agree. But this is a bit of a overthinking at this point in time, IMHO. We don't have GDM and the other stuff yet. We can update the naming when the need arrives for it.

Reply:
Well since we know what we'll have more or less, I'd pick the right name from the beginning to reduce the clutter and extra diff in future MRs.

Reply:
maybe `TestCLIIntegration`?

Reply:
Oh, I'm good with this... I just can't resolve conversations myself :-(